include/SimpleTextFloat.h

```diff
@@ -31,6 +31,7 @@
 #include "lmms_export.h"
 
 class QLabel;
+class Qtimer;
```

Comment:
Should be ``QTimer``
```suggestion
class Qtimer;
```